This PR adds complete updated website structure.

Koushith opened the pull request:
## This PR adds the following changes
- changed the color and added the logo as per Figma mockup.
- Changed the font from default one to 'JoseFin Sans'.
- Added FAQ section without installing any Dependencies.
- Partial Responsiveness

###  Pending: 
- [x] Add icons/Team images and fix alignments.
- [x] Contact form integration.
- [x] complete mobile responsiveness



<br><br>
![2020-08-14 03-00-11](https://user-images.githubusercontent.com/30016242/90192041-b60d6c80-dddf-11ea-96b2-44e6a31a0111.gif)

Koushith committed:
adds all components as per the UI mockup

Koushith committed:
adds contact form

koshikraj commented on src/components/Faqs/Faq.js:
```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
```
This is a good one :+1:

koshikraj commented on src/components/Faqs/faqComponent.js:
```diff
@@ -0,0 +1,17 @@
+      key={index}
+      onClick={() => toggleFAQ(index)}
+    >
+      <div className='faq-question cursor-pointer'>{faq.question}</div>
```
Can we add a down arrow or plus icon to the right of this div?

Koushith replied:
oh yeah, I don't know how did I miss that. will add it right away.

koshikraj commented on src/components/Hero.js:
```diff
@@ -2,24 +2,24 @@ import React from 'react';
 const Hero = () => {
   return (
-    <section className='text-gray-700 body-font'>
+    <section className=' font-body bg-secondary-blue h-screen'>
```
The Hero section has a gab at the bottom. Either we need to fill it with something or reduce the size.

Koushith replied:
may be we can reduce the Viewport size to 80vh.

abhinav-anshul replied:
try viewport, could be helpful

koshikraj commented on src/components/Description.js:
```diff
@@ -29,14 +31,18 @@ function Description() {
+                  Configurable Rules
                 </h2>
-                <p className='leading-relaxed'>Discover</p>
+                <p className='text-gray-700 mt-4 flex-1'>
```
I think we need to keep the sizes fixed for the cards. It looks uneven. Thoughts on this?

Koushith replied:
yes, it's uneven and the card last one is left aligned. this was because of lengthy class names. Flex property was not working .will add the icons clean this up.

Koushith committed:
adds different section, integrated contact form and more

Koushith commented:
## This commit includes below changes

- [x] Fixed the height (reduces to 80vh) on a larger screen.
- [x] Integrated Contact submission form.
- [x] Fixed Nav router and implemented smooth scrolling.
- [x] Fixed the uneven size of Div's
- [x] Mobile Responsiveness.
- [x] Arrow icons on FAQ's

## Todos:
- [ ] Google Analytics.
- [ ] Hide the main menu on smaller devices.
- [ ] Update all placeholder images and pixel-perfect the mobile responsiveness.

Koushith committed:
adds websafe font

Koushith committed:
Fiexed absolute link issue

koshikraj reviewed (approved):
LGTM. A few images and placeholder content needs to replaced.
